docs/curate-audio/tutorials/beginner.md

```diff
@@ -178,3 +270,113 @@ Example output entry:
     "duration": 2.3
 }
 ```
+
+**Field descriptions:**
+- `audio_filepath`: Absolute path to audio file
+- `text`: Ground truth transcription from FLEURS
+- `pred_text`: ASR model prediction
+- `wer`: Word Error Rate percentage (0.0 = perfect match)
+- `duration`: Audio duration in seconds
+
+**Analyzing results:**
+```bash
+# Count filtered samples
+cat /data/fleurs_output/result/*.jsonl | wc -l
+
+# View first 5 samples
+head -n 5 /data/fleurs_output/result/*.jsonl | jq .
+
+# Calculate average WER
+cat /data/fleurs_output/result/*.jsonl | jq -r '.wer' | awk '{sum+=$1; count+=1} END {print "Average WER:", sum/count "%"}'
+```
+
+**Using Python:**
+```python
+import json
+import pandas as pd
+from pathlib import Path
+
+# Load results
+result_files = list(Path("/data/fleurs_output/result").glob("*.jsonl"))
+data = []
+for file in result_files:
+    with open(file, 'r') as f:
+        for line in f:
+            data.append(json.loads(line))
+
+df = pd.DataFrame(data)
+
+# Summary statistics
+print(f"Total samples: {len(df)}")
+print(f"Average WER: {df['wer'].mean():.2f}%")
+print(f"Average duration: {df['duration'].mean():.2f}s")
+print(f"WER range: {df['wer'].min():.2f}% - {df['wer'].max():.2f}%")
+```
+
+## Troubleshooting
+
+### Common Issues
+
+**GPU out of memory:**
+```
+RuntimeError: CUDA out of memory
+```
+**Solution:** Reduce batch size or use a smaller ASR model:
+```python
+pipeline.add_stage(
+    CreateInitialManifestFleursStage(...).with_(batch_size=2)  # Reduce from 4 to 2
+)
+```
+
+**Dataset download fails:**
+```
+ConnectionError: Failed to download FLEURS dataset
+```
+**Solution:** Check internet connection and retry. The stage will resume from where it left off.
+
+**No GPU available:**
+```
+RuntimeError: No CUDA GPUs are available
+```
+**Solution:** Ensure CUDA is installed and GPU is accessible:
+```bash
+nvidia-smi  # Check GPU availability
+python -c "import torch; print(torch.cuda.is_available())"
+```
+
+**Model download fails:**
+```
+OSError: Model 'nvidia/stt_...' not found
+```
+**Solution:** Verify model name is correct and you have internet access. Check available models at [NGC Catalog](https://catalog.ngc.nvidia.com/).
+
+### Performance Optimization
+
+- **Increase batch size** for faster processing (if GPU memory allows)
+- **Use multiple GPUs** by setting `Resources(gpus=2.0)` or higher
+- **Process subset of data** by using `split="dev"` (smaller than "train")
+- **Skip ASR inference** if you already have transcriptions (remove InferenceAsrNemoStage)
```

Comment:
If you remove ASR inference, then what quality will you assess? 
We can remove this line or rephrase: 
**Skip ASR inference** if you already have both predicted and target transcriptions. (remove InferenceAsrNemoStage)